src/keyboard.js

```diff
@@ -0,0 +1,34 @@
+//globals $
+  export default function() {
+    var enter = $(`#write`);
+
+    $(`.keyboard-row .keyboard-button`).click(function() {
+      var char = $(this).html();
+
+      enter.html(enter.html() + char);
+    });
+
+    $(`.keyboard-row .keyboard-button_tab`).click(function() {
+      var char = `\t`;
+
+      enter.html(enter.html() + char);
+    });
+
+    $(`.keyboard-row .keyboard-button_space`).click(function() {
+      var char = ` `;
+
+      enter.html(enter.html() + char);
+    });
+
+    $(`.keyboard-row .keyboard-button_return`).click(function() {
+      var char = `\n`;
+
+      enter.html(enter.html() + char);
+    });
+
+    $(`.keyboard-row .keyboard-button_delete`).click(function() {
+      var char = `a`;
+
+      enter.html(enter.html() - char);
```

Comment:
Subtracting a string from anything will produce `NaN`.

You should instead look at using the `substring` or `slice` methods on strings so that you can get a piece of the string.

For instance:

```
var str1 = 'The morning is upon us.';
var str2 = str1.slice(4, -2);

console.log(str2); // OUTPUT: morning is upon u
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice

You'll need to play with the values.